Makefile

```
@@ -303,3 +304,6 @@ test-e2e-helm: image-build-helm
 
 test-integration: ## Run integration tests
 	./hack/tests/integration.sh
+
+olm-manifests:
```

We have `gen-cli-doc` and  `gen-changelog` So: 

```suggestion
gen-olm-bindata: ## Generate bindata with the OLM manifests
```